### ISSUE TITLE
b.cesm3_cam058_mom_e.B1850WscMOM.ne30_L58_t061.camdev_cice5.027a

### ISSUE THREAD

**Issue by gustavo-marques** [NCAR/amwg_dev#182]
**Description:**

Same as [026a](https://github.com/NCAR/amwg_dev/issues/178) with a hybrid vertical coordinate system in the ocean model

In `user_nl_mom` we set:

```
INPUTDIR = "INPUT"
NK = 75
REGRIDDING_COORDINATE_MODE = "HYCOM1"
INTERPOLATION_SCHEME = "P1M_H2"
REMAPPING_SCHEME = "PPM_H4"
BOUNDARY_EXTRAPOLATION = True
ALE_COORDINATE_CONFIG = "HYBRID:hybrid_75layer_zstar2.50m-2020-11-23.nc,sigma2,dz"
REGRID_COMPRESSIBILITY_FRACTION = 0.01
MAXIMUM_INT_DEPTH_CONFIG = "FNC1:5,8000.0,1.0,.01"
MAX_LAYER_THICKNESS_CONFIG = "FNC1:400,31000.0,0.1,.01"
COORD_CONFIG = "ALE"
```
Directory `INPUT` must be copied to the $RUNDIR with appropriate symbolic links in it.

**Case directory:**
Locally (if still available):
/glade/p/cesmdata/cseg/runs/cesm2_0/b.cesm3_cam058_mom_e.B1850WscMOM.ne30_L58_t061.camdev_cice5.027a

On github:

**Sandbox:**
Locally (if still available):
/glade/work/hannay/cesm_tags/cesm3_cam6_3_058_MOM_e

On github:
hash: 8f70c08

**Diagnostics:**
AMWG diags (if available)
[https://webext.cgd.ucar.edu/BWsc1850MOM/$CASE/atm/](https://webext.cgd.ucar.edu/BWsc1850MOM/$CASE/atm/)

**Contacts:**
@gustavo-marques, @cecilehannay, @JulioTBacmeister, @fobryan3, @adamrher

**Comment by justin-richling** [comment]
Quick ADF vs 026a for years 17-27 can be found [here](https://webext.cgd.ucar.edu/BWsc1850MOM/b.cesm3_cam058_mom_e.B1850WscMOM.ne30_L58_t061.camdev_cice5.027a/atm/b.cesm3_cam058_mom_e.B1850WscMOM.ne30_L58_t061.camdev_cice5.027a_17_27_vs_b.cesm3_cam058_mom_e.B1850WscMOM.ne30_L58_t061.camdev_cice5.026a_17_27/).

**Comment by justin-richling** [comment]
Timeseries:

![TimeSeries_ANN](https://user-images.githubusercontent.com/56696811/192566467-ed29c65c-4707-4465-b1c6-219155f08aa3.png)

**Comment by swrneale** [comment]
Yikes, this is depressing. We're just delaying icing almost no matter what. I wonder if we run out 23, and even though it's warm, use that as an initial condition for subsequent runs.

**Comment by gustavo-marques** [comment]
Interesting how RESTOM oscillates in 27a. No other run shows such behavior.

**Comment by justin-richling** [comment]
27a is being plotted with only the yearly average while the others are 5-yr rolling, would that explain the appearance of the oscillating behavior?

**Comment by gustavo-marques** [comment]
Yes, that explains! Thanks, @justin-richling.

**Comment by JulioTBacmeister** [comment]
We decided to extend 25a which has somewhat more acceptable RESTOM than
23.  Let's see where that goes.  I think we need to reassess our strategy
this Friday. We also need to look more closely at the 100 years of 26a and
26b despite the Lab Sea freeze.  What does ENSO look like ... etc.  ?

**Comment by gustavo-marques** [comment]
Sounds good. I am going to stop 27a.

**Comment by JulioTBacmeister** [comment]
Hmmm is convection totally dead everywhere???

**Comment by gustavo-marques** [comment]
@JulioTBacmeister: the convection is the Lab Sea died after ~ 7 years and did not recover (it's at year 30 now). I am happy to continue the run if you think it's worth it.

**Comment by cecilehannay** [comment]
@adamrher: you can run directly the CVDP from the ADF. I will ask @justin-richling to give it a try.

**Comment by JulioTBacmeister** [comment]
@gustavo-marques the feeling in the Ocean section has always been that once dead there is no way for the N Atlantic convection to recover.  Is this worth revisiting with MOM?  I leave it to you experts.   
Another question: Is convection dead everywhere in 27a, i.e., Irminger Sea ... further east ...?

**Comment by JulioTBacmeister** [comment]
To start things off ... Time series of SST in Nino34 for 26a and 26b.
![image](https://user-images.githubusercontent.com/9221710/192620613-b6e5e6fc-f50d-4c24-ab42-ea4da028f8cd.png)
![image](https://user-images.githubusercontent.com/9221710/192620647-d183faca-c9ab-41ea-b8b3-55aa67d1545e.png)

**Comment by gustavo-marques** [comment]
@JulioTBacmeister: no, convection still occurs elsewhere. The plot below shows winter mean MLDs averaged between years 17-27.
![b cesm3_cam058_mom_e B1850WscMOM ne30_L58_t061 camdev_cice5 027a_MLD_winter](https://user-images.githubusercontent.com/11339137/192624272-80bb1cfd-01e1-493d-bf46-1d18520f9d15.png)

**Comment by JulioTBacmeister** [comment]
So, is 27a still "viable"? If AMOC is being forced in different regions,
but is still there is that OK?  Perhaps with the biases we see in 1 degree
ocean models we should accept that convection may happen in different
locations than in reality.  It's interesting that the mixed layer depth in
27a is bigger than estimated in several places.

**Comment by swrneale** [comment]
Not to repeat exacty what you showed Julio, but ENSO looks pretty darn good, especially early on in the simulations.

![image](https://user-images.githubusercontent.com/8239194/192781993-bc14d713-a0c6-4cd2-9c2b-3acb4ffa40f3.png)![image](https://user-images.githubusercontent.com/8239194/192782715-5831e2e1-de25-4f7f-9380-260efa4a2692.png)

**Comment by JulioTBacmeister** [comment]
Thanks Rich, This is interesting.  Is it worth looking at ENSO in some of our earlier runs even though they are shorter?

**Comment by swrneale** [comment]
Sure. I think if we have 30 years, we can at least have a sense of the period and maybe autocorrelation. With magnitude you're never quite sure. Is it worth starting a 'discussion' of non-standard. Which runs? @cecilehannay @JulioTBacmeister is it worth starting a 'discussion' of non standard diagnostics for some key runs (MJO, blocking, monsoon etc.). I have a few scripts for these, same as CESM2 development.

**Comment by JulioTBacmeister** [comment]
Why don't we look at 23, 25a and 25b.  And I like the idea of setting up a
new discussion on github.  We can call it something like "Atmospheric means
and variability in coupled runs" ...? Note also: @Justin Richling
***@***.***>  is trying to get the CVDP analyses up for the 26's.

Anyone know how to move our recent comments and figures from the 27a issues
(a bad place) to this new discussion?

**Comment by gustavo-marques** [comment]
@JulioTBacmeister: sorry for the delay in responding to your question. I think 27a is not "viable". AMOC time series at both 26 N and 45 N are declining very rapidly, see figures below (please ignore the years).
![b cesm3_cam058_mom_e B1850WscMOM ne30_L58_t061 camdev_cice5 027a_MOC_26N_time_series](https://user-images.githubusercontent.com/11339137/192893913-3efb7daf-8809-4bcf-90a8-bb291c9928e8.png)
![b cesm3_cam058_mom_e B1850WscMOM ne30_L58_t061 camdev_cice5 027a_MOC_45N_time_series](https://user-images.githubusercontent.com/11339137/192893932-3fb8145c-9446-4d89-b0bd-6481a1c75835.png)
